modules/dynamic-plugins/proc-export-third-party-plugins-rhdh.adoc

```diff
@@ -62,7 +62,7 @@ The following is an example of default `scalprum` configuration:
 "scalprum": {
   "name": "<package_name>",  // The Webpack container name matches the NPM package name, with "@" replaced by "." and "/" removed.
   "exposedModules": {
-    "PluginRoot": "./src/index.ts"  // The default module name is "PluginRoot" and doesn't need explicit specification in the app-config.yaml file.
+    "PluginRoot": "./src/index.ts"  // The default module name is "PluginRoot" and doesn't need explicit specification in the {my-app-config-file} file.
```

Comment:
You introduce an attribute in a code block that had none. Do not forget to tell asciidoctor to substitute the attributes in the code block. See https://docs.asciidoctor.org/asciidoc/latest/subs/apply-subs-to-blocks/#incremental

`[source,json,subs="+attributes"]`